mycli/main.py

```diff
@@ -818,7 +827,22 @@ def cli(database, user, host, port, socket, password, dbname,
                 confirm_destructive_query(stdin_text) is False):
             exit(0)
         try:
-            mycli.run_query(stdin_text, table_format=table)
```

Comment:
@tsroten You're right. I'll refactor this out. Maybe adding a new parameter called `new_line` to the `run_query` function?